### DISCUSSION TITLE
Issue with Front-End Updates Affecting Page Content from Database

### DISCUSSION THREAD

**Discussion by kaansenol** [OFFLINE-GmbH/oc-boxes-support#151] [Help]
Hi,

I’m currently working on a project using October CMS along with the Offline Boxes plugin to manage page content. I’m facing an issue where every time my front-end developer pushes her updates to the Git repository and I pull those updates, the page contents on the website change unexpectedly.

The content is stored in the database, so I’m unsure why pulling front-end updates is affecting the page content. It seems like the database-driven content should remain intact regardless of front-end changes. Has anyone else experienced this, or is there a specific setting in the Offline Boxes plugin that I might be overlooking to prevent this behavior?

Any guidance or suggestions on how to resolve this would be greatly appreciated!

Thank you.

**Comment by tobias-kuendig** [comment]
> I’m facing an issue where every time my front-end developer pushes her updates to the Git repository and I pull those updates, the page contents on the website change unexpectedly.

Boxes stores all content in the `offline_boxes_boxes` page. These contents are not affected by the frontend theme. However, if your frontend dev changes any of the boxes partials in the theme, it might be that you see the page visually different. However, the text displayed (= the data) should still remain the same.
